app/controllers/api/v1/admin/articles_controller.rb

```diff
@@ -14,8 +14,9 @@ def create
 
   def update
     authorize(current_user)
-
-    Article.update(params[:id], published: params[:article][:published], publisher_id: current_user.id)
+    Article.update(params[:id], published: params[:published], publisher_id: current_user.id)
+    article = Article.find(params[:id])
+    render json: article
```

Comment:
why do you want to send back the article?

Reply:
```
render head: :ok 
```

Reply:
fixed